src/content/updates/whats-new/whats-new.mdx

```diff
@@ -1,28 +1,223 @@
+- Disabled checkbox to show check in dark themes
+- Content switcher to use `$disabled-0*` color tokens for dark themes
+- Inline dropdown background and disabled colors in dark themes
+- Skeleton color tokens to work in all themes
+- Theme `$highlight` token in dark themes
+- Theme `$hover-primary-text` token in dark themes
+- Theme `$hover-ui` token in g90 theme
+
+#### Features:
+
+- Emit custom properties for `@carbon/themes` color tokens
+
+### Icon updates
+
+#### Bug fixes 🐛 
+
+* battery--low.svg
+* ibm-security.svg
+* logo--slack.svg
+* logout.svg
+* mobile--add.svg
+* mobile--check.svg
+* mobile--download.svg
+* mobile.svg
+* popup.svg
+* text--strikethrough.svg
+* mammogram.svg
+
+#### Enhancements 🛠 
+
+- Bespoke icons added at the following sizes: `20/time.svg`
+- `@carbon/icons-react` now supports an entrypoint `index.js`, you no longer are required to specify the path for an icon
+
+#### New ✨ 
+
+* language.svg
+* laptop.svg
+* account.svg
+* branch.svg
+* enterprise.svg
+* fog.svg
+* help-desk.svg
+* no-image.svg
+* no-ticket.svg
+* pause--filled.svg
+* play--filled--alt.svg
+* service-desk.svg
+* skip--back--filled.svg
+* skip--forward--filled.svg
+* stop--filled--alt.svg
+* audio-console.svg
+* battery--empty.svg
+* battery--quarter.svg
+* carbon.svg
+* category--add.svg
+* category--and.svg
+* category--new-each.svg
+* category--new.svg
+* category.svg
+* concept.svg
+* fork.svg
+* hurricane.svg
+* mobile--landscape.svg
+* partly-cloudy.svg
+* sunny.svg
+* sunset.svg
+* tablet--landscape.svg
+* tablet.svg
+* term.svg
+* text-link--analysis.svg
+* text-link.svg
+* text-mining--applier.svg
+* text-mining.svg
+* tree-view--alt.svg
+* type-pattern.svg
+* types.svg
+
+#### Removed ☠️ 
+
+* mammogram--stacked.svg
+
+### UI shell enhancements
+
+#### New ✨ 
+
+- Guidance for UI Shell Navigation patterns
+- UI Shell Sketch template and symbols added to the Sketch kit.
+- Rail functionality for UI shell left panel.
+- Responsive behavior for the header with nav links.
+
+### Support and component enhancements
+
+#### Bug fixes 🐛 
+
+- Accessibility updates to Tooltips
+
+   * Definition tooltip WCAG 2.1 compliance (#3151)
+   * Interactive tooltip html updates for a11y (#3148)
+- Update toggle labels to be accessible (#2885)
+- Accessibility updates for underlying Listbox (#3139)
+- Data Table updates
+
+  * data table merge sticky header style (#2965)
+  * apply persistent styles to data table (#3311)
+  * fix data table clear input button functionality (#3285)
+  * reset expand button height in short table (#3057)
+- Accessibility updates for Overflow Menu
+
+#### Enhancements 🛠 
+
+* `<Search>` component `light` variant (#3230)
+
+#### New ✨ 
+
+* add icon-only button to React library (#3368)
+* inline-notification action button with click handler (#3283)
+
+### Design kit
+
+#### Bug fixes 🐛 
+
+- Icon button: added disabled state [#121](https://github.com/carbon-design-system/carbon-design-kit/issues/121)
+- Dropdown:
+
```

Comment:
Same as before

Reply:
For some reason, the design kit section bullet points are spaced out different than the rest of the bullet points. 
<img width="769" alt="Screen Shot 2019-08-08 at 10 47 19 AM" src="https://user-images.githubusercontent.com/32556167/62717726-0df1e600-b9ca-11e9-8f6b-2e4eb995f47e.png">

Not sure why that is happening but I think we should be consistent with how we space things.

Reply:
I explained above, but this is an issue with the way the site handles nested unordered lists. It breaks if you don't space it out. No idea why this is happening, I'd rather it not but we're stuck with it until we fix it